mmv1/templates/terraform/examples/organization_security_policy_association_basic.tf.tmpl

```diff
@@ -1,18 +1,15 @@
 resource "google_folder" "security_policy_target" {
-  provider     = google-beta
   display_name = "tf-test-secpol-%{random_suffix}"
   parent       = "organizations/{{index $.TestEnvVars "org_id"}}"
   deletion_protection = false
 }
 
 resource "google_compute_organization_security_policy" "policy" {
-  provider = google-beta
   display_name = "tf-test%{random_suffix}"
   parent       = google_folder.security_policy_target.name
 }
 
 resource "google_compute_organization_security_policy_rule" "{{$.PrimaryResourceId}}" {
-  provider = google-beta
```

Comment:
Also, this test `TestAccComputeOrganizationSecurityPolicyAssociation_organizationSecurityPolicyAssociationBasicExample` failed in the GA CI, as `google_compute_organization_security_policy_rule` is not in Terraform GA provider yet. Can you please remove this resource from this file? Thanks.